<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>876</width>
    <height>520</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>876</width>
    <height>520</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>890</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(48, 84, 44, 255), stop:0.0340909 rgba(14, 14, 14, 255), stop:0.971591 rgba(50, 12, 12, 255), stop:1 rgba(173, 24, 24, 255));

selection-background-color: rgb(155, 84, 255);
border-left-color: rgb(0, 255, 0);
color: rgb(255, 255, 255);


QPushButton#shotBuild {
    background-color: yellow;
}

QPushButton#shotBuild:hover {
    background-color: rgb(224, 255, 0);
}

QPushButton#shotBuild:pressed {
    background-color: rgb(224, 0, 0);     
}



</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>791</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Animation</string>
   </property>
   <widget class="QListWidget" name="anim">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>771</width>
      <height>101</height>
     </rect>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::ExtendedSelection</enum>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>791</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Layout</string>
   </property>
   <widget class="QListWidget" name="layout">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>771</width>
      <height>101</height>
     </rect>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::ExtendedSelection</enum>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>791</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Matchmove</string>
   </property>
   <widget class="QListWidget" name="matchmove">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>771</width>
      <height>101</height>
     </rect>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::ExtendedSelection</enum>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="shotBuild">
   <property name="geometry">
    <rect>
     <x>664</x>
     <y>462</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Shot Build</string>
   </property>
  </widget>
  <widget class="QPushButton" name="e_anim">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>70</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>E</string>
   </property>
  </widget>
  <widget class="QPushButton" name="e_lay">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>220</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>E</string>
   </property>
  </widget>
  <widget class="QPushButton" name="e_match">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>350</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>E</string>
   </property>
  </widget>
  <widget class="QPushButton" name="view_layout">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Preview Layout</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="frame_range">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>470</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Set frame range</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../../../../../../Pipeline/houdini/houdini16.0/scripts/QtQRC/build_shot_qrc.qrc"/>
 </resources>
 <connections/>
</ui>
